packages/mwp-app-render/src/util/seoHelper.jsx

```diff
@@ -0,0 +1,331 @@
+import React from 'react';
```

Comment:
This was copied with next to no refactoring, though `convertToLocalTime` was added.

Reply:
I like that idea